src/main/java/com/adyen/service/Payment.java

```diff
@@ -99,7 +100,7 @@ public PaymentResult authorise3D(PaymentRequest3d paymentRequest3d) throws Excep
      * @return PaymentResult
      * @throws Exception Exception
```

Comment:
Hello @H4rryp0tt3r !

Could you also fix the java code for `@throws` in this method? It seems our travis build is failing because javadocs style.

Regards,
Renato
Adyen

Reply:
Hello @martinsrenato!

I've added those changes and force pushed as it made sense to keep it in the same commit. However travis didn't pick those.

Thanks,
Nagesh